src/templates/_select.twig

```diff
@@ -1,11 +1,126 @@
-{% if name is defined and name %}
-	<input type="hidden" name="{{ name }}" value="">
-{% endif %}
-<div class="multiselect">
-	<select name="{{ name }}[]" multiple>
-		<option value="">{{ "None"|t }}</option>
-		{% for option in products %}
-			<option value="{{ option.productId }}"{% if option.productId in value %} selected{% endif %}>{{ option.label }}</option>
-		{% endfor %}
-	</select>
-</div>
\ No newline at end of file
+{% set now = "now"|date("U") %}
+
+<div class="{{ wrapper_class }}_{{ now }}">
+    <div style="margin-bottom: 5px; margin-top: 5px;">
+        {% if selected_only_class is defined and selected_only_class %}
+            <span style="float: left;">
+                <input type="checkbox" class="{{ selected_only_class }}_{{ now }}" value=""><span style="margin-left: 5px;">{{ 'Show selected only'|t }}</span>
+            </span>
+        {% endif %}
+
+        {% if clear_selected_class is defined and clear_selected_class %}
+            <span style="float: right;">
+                <button class="{{ clear_selected_class }}_{{ now }}">{{ 'Clear selected'|t }}</button>
+            </span>
+        {% endif %}
+    </div>
+
+    {% if name is defined and name %}
+        <input type="hidden" name="{{ name }}" value="">
+    {% endif %}
+    <div class="multiselect">
+        <select style="width: 100%; height: 20rem;" class="{{ name }}_{{ now }}" name="{{ name }}[]" multiple>
+            <option value="">{{ "None"|t }}</option>
+            {% for option in products %}
+                <option value="{{ option.productId }}"{% if option.productId in value %} selected{% endif %}>{{ option.label }} ({{ option.sku }})</option>
```

Comment:
I actually prefer knowing there is no sku for my team, because we usually want to fix these. I guess they still know without the parens but this makes it more obvious and makes our data people want to fix it.

Reply:
Might be true for your team, but since this is a plugin which is used by a wider variete of users and not all will use/maintain this ski field. Therefore please make it optional as suggested.